### PR TITLE
USAGOV-1146-PR template updated

### PR DESCRIPTION
**Pull request by YaritzaGarcia**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1146

## Description

Update the current PR template based on the [PR Template Brainstorm](https://docs.google.com/document/d/1dj0PptulgFOzihwEM9DQr7PC_oaqYyXUVJ9f8-VyuhM/edit#heading=h.s1w6hh4mcjsb).

## Checklist for the Developer

- [x] A link to the JIRA ticket has been added above.
- [x] The JIRA ticket identifies the desired result of this change.
- [x] The JIRA ticket contains clear acceptance criteria.
- [x] The JIRA ticket contains clear testing steps.
- [x] The JIRA ticket contains a description of "Done".
- [x] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [ ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

### CODE REVIEW

**Review by IsabelLaurenceau — CHANGES_REQUESTED** [review]
So far so good, I only have a few requests. 
I think 
### Requires New Content/Config
- [ ] Yes
- [ ] No
 
should be two separate checkbox items since they are rectified differently i.e 
### Requires New Config
- [ ] Yes
- [ ] No

### Requires New Content
- [ ] Yes
- [ ] No

For the Post PR Approval Instructions since there are multiple ways to approve on CircleCi I would just keep instructions 1 & 2 and aded a 3rd that says "build and deploy." Also keeps it shorter so less likely to be skipped over. 

Also just want to mention the comments don't come up in the markdown. It looks like you kinda want them too based on how you wrote them so maybe just change the ones you want to appear to normal text. You can italicizes them to show it's like "instructions for the section"

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -4,24 +4,55 @@
...
-- [ ] Following AC and Testing Steps verify changes work as expected.
-- [ ] There are no known side-effects outside of expected behavior.
+<!--- Summarize the chages made in this pull request, not what it's for. -->
+
+## Type of Changes
+<!--- Put an `x` in all the boxes that apply. -->
+- [ ] New Feature
+- [ ] Bugfix
+- [ ] Frontend (Twig, Sass, JS)
+  - Add a screenshot on how it should look like
```
Comment (akf) [inline .github/pull_request_template.md]: Text tweak: "Add screenshot showing what it should look like"

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -4,24 +4,55 @@
...
+- [ ] New Modules (requires rebuild)
+- [ ] Infrastructure
+  - [ ] CMS
+  - [ ] WAF
+  - [ ] Egress
+  - [ ] Tools
+- [ ] Other
+
+## Testing Instructions
+<!-- This instructions are d ifferent from “testing instructions” in Jira – those are typically for Content/UX stakeholders -->
```
Comment (akf) [inline .github/pull_request_template.md]: Remove stray space in "d ifferent"

Comment (jacobaaronyeager) [inline .github/pull_request_template.md]: looks like there is an errant space in the word different: d ifferent

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -4,24 +4,55 @@
...
+<!-- Not “see Jira” – if they are really the same, copy and paste. -->
+
+### Requires New Content/Config
+- [ ] Yes
+- [ ] No
+
+### Validation Steps
+- [ ] Test instruction 1
```
Comment (akf) [inline .github/pull_request_template.md]: These should probably just be bullets, not checkboxes.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -4,24 +4,55 @@
...
+- [ ] Test instruction 2
+- [ ] Test instruction 3
+
+## Security Review
+<!-- Checkboxes to indicate need for review -->
+
+- [ ] Adds/updates software (including a library or Drupal module)
+- [ ] Communication with external service
+- [ ] Chnges permissions or workflow
```
Comment (akf) [inline .github/pull_request_template.md]: Chnges -> Changes

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -4,24 +4,55 @@
...
+- Security review complete or not required
+
+## Post PR Approval Instructions
+<!-- Follow the following steps as soon as you merge the new changes. -->
+1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
+2. Find the commit of this pull request.
+3. Click the arrow next to the "build-and-deploy" section.
+4. Under the "Jobs" section the first thing that should appear is "approve-build-and-push-container". Press the thumbs up icon, which appears next to "approve-build-and-push-container", to approve the build in CircleCI.
+5. Wait for this process to finish to continue with the next step. You'll know it's done when the check mark icon appears on the left side of "build-and-push-container."
+6. Press the thumbs up icon, which appears next to "approve-dev-deployment", to deploy the changes to dev. Once the deployment to cloudgov-dev is complete, [cms-dev.usa.gov](http://cms-dev.usa.gov/) will generate the new code and, 20 to 30 minutes later, the changes They will be made visible at [beta-dev.usa.gov](http://beta-dev.usa.gov/).
```
Comment (jacobaaronyeager) [inline .github/pull_request_template.md]: Extra "They" in the last clause: the changes **They** will be made visible

Comment (akf) [inline .github/pull_request_template.md]: remove word "They".

**Review by jacobaaronyeager — CHANGES_REQUESTED** [review]
Looks amazing!  A couple small changes, let me know when you push and I'll approve :)

File: .github/pull_request_template.md (hunk 5 above)
Comment (akf) [reply .github/pull_request_template.md]: Let's add a final instruction to update the Jira ticket. My suggestion: 

7. Update the Jira ticket: change the status of the ticket to *Review in Test* and add a comment. State whether the change is already visible on  [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

**Review by akf — CHANGES_REQUESTED** [review]
Thanks! I proofread and made a couple of suggestions.

**Review by akf — APPROVED** [review]

**Review by jacobaaronyeager — APPROVED** [review]

**Review by IsabelLaurenceau — APPROVED** [review]

**Pull request merged**